### PR TITLE
rewriting templates/filepath/examples as code

### PR DESCRIPTION
**Pull request by Remi-Gau**
An attempt to solve issue #5 .
Also fixed a small typo in one DICOM tag

### CODE REVIEW

File: specification.md [hunk 1]
```diff
@@ -331,40 +337,46 @@ Files for a particular participant can exist only at participant level directory
...
->> sub-01\_task-xyz\_acq-test1\_bold.json  
->> anat/  
->> func/  
->>> sub-01\_task-xyz\_acq-test1\_run-1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon2\_bold.nii.gz  
+
+```markdown
+sub-01/  
+	sub-01_task-xyz_acq-test1_bold.json
```
Comment (chrisfilo) [inline specification.md]: could you change this to 4 spaces?

File: specification.md [hunk 2]
```diff
@@ -331,40 +337,46 @@ Files for a particular participant can exist only at participant level directory
 Example 1: Two JSON files at same level that are applicable for NIfTI
 file.
 
+```markdown
```
Comment (chrisfilo) [inline specification.md]: Could you remove syntax highlight from this section?

File: specification.md [hunk 3]
```diff
@@ -331,40 +337,46 @@ Files for a particular participant can exist only at participant level directory
...
 
-> sub-01/  
->> anat/  
->> func/        
->>> sub-01\_task-xyz\_acq-test1\_run-1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_run-2\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon2\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_bold.json
+```markdown
```
Comment (chrisfilo) [inline specification.md]: Could you remove syntax highlight from this section?

File: specification.md [hunk 4]
```diff
@@ -331,40 +337,46 @@ Files for a particular participant can exist only at participant level directory
...
 Case 2:  Multiple json files at different levels for same task and acquisition parameters  
-> sub-01/  
->> sub-01\_task-xyz\_acq-test1\_bold.json  
->> anat/  
->> func/  
->>> sub-01\_task-xyz\_acq-test1\_run-1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon1\_bold.nii.gz  
->>> sub-01\_task-xyz\_acq-test1\_rec-recon2\_bold.nii.gz  
+
+```markdown
```
Comment (chrisfilo) [inline specification.md]: Could you remove syntax highlight from this section?

File: specification.md [hunk 5]
```diff
@@ -571,23 +583,25 @@ Tabular files MAY be optionally accompanied by a simple data dictionary in a JSO
...
-&nbsp;&nbsp;&nbsp; "3": "Student at university",  
-&nbsp;&nbsp;&nbsp; "4": "Has degree from university"  
-&nbsp;&nbsp; }  
-&nbsp;},  
-&nbsp;"bmi": {  
-&nbsp;&nbsp; "LongName": "Body mass index",  
-&nbsp;&nbsp; "Units": "kilograms per squared meters",  
-&nbsp;&nbsp; "TermURL": "http://purl.bioontology.org/ontology/SNOMEDCT/60621009"  
-&nbsp;}  
+```markdown
```
Comment (chrisfilo) [inline specification.md]: Could you switch syntax highlight to `JSON`?

File: specification.md [hunk 6]
```diff
@@ -596,11 +610,12 @@ JavaScript Object Notation (JSON) files MUST be used for storing key/value pairs
 files MUST be in UTF-8 encoding.
 
 ### 4.3.1 Example:
-
+``` markdown
```
Comment (chrisfilo) [inline specification.md]: Could you switch syntax highlight to JSON?

File: specification.md [hunk 7]
```diff
@@ -643,37 +658,40 @@ This is an example of the folder and file structure. Because there is only one s
...
-    -   Additional files and folders containing raw data may be added as needed for special cases.  They should be named using all lowercase with a name that reflects the nature of the scan (e.g., “calibration”).  Naming of files within the directory should follow the same scheme as above (e.g., “sub-control01_calibration_Xcalibration.nii.gz”)  
--   code  
-    -   deface.py  
--   derivatives  
-    -   README  
--   participants.tsv  
--   dataset\_description.json  
--   README  
--   CHANGES  
+``` markdown
```
Comment (chrisfilo) [inline specification.md]: Could you remove syntax highlight from here?

File: specification.md [hunk 8]
```diff
@@ -728,18 +746,19 @@ The file dataset\_description.json is a JSON file describing the dataset. Every
   </tbody>
 </table>
 
-Example:  
+Example: 
+``` markdown
```
Comment (chrisfilo) [inline specification.md]: Could you switch syntax highlight to JSON?

File: specification.md [hunk 9]
```diff
@@ -990,14 +1009,16 @@ version of [DICOM ontology terms](https://scicrunch.org/scicrunch/interlex/dashb
...
 Template:
-
-sub-&lt;participant\_label&gt;/[ses-&lt;session\_label&gt;/]  
-> anat/  
->> sub-&lt;participant\_label&gt;[\_ses-&lt;session\_label&gt;][\_acq-&lt;label&gt;][\_ce-&lt;label&gt;][\_rec-&lt;label&gt;][\_run-&lt;index&gt;\]\_&lt;modality\_label&gt;.nii[.gz]
-
-sub-&lt;participant\_label&gt;/[ses-&lt;session\_label&gt;/]  
-> anat/
->> sub-&lt;participant\_label&gt;[\_ses-&lt;session\_label&gt;][\_acq-&lt;label&gt;][\_ce-&lt;label&gt;][\_rec-&lt;label&gt;][\_run-&lt;index&gt;][\_mod-&lt;label&gt;]\_defacemask.nii[.gz]
+``` markdown
```
Comment (chrisfilo) [inline specification.md]: Could you remove syntax highlight?

**Comment by chrisfilo** [comment]
Actually I will handle this - hold on.

**Comment by Remi-Gau** [comment]
I also saw that there bits of filepaths sometimes imbeded in text sections. Do we want those to appear as code too?

**Comment by chrisfilo** [comment]
We covered this in recent PRs. Apologies for wasted effort. Early days...

**Comment by Remi-Gau** [comment]
No worries: this was anyway a good exercise for me to finally start getting better acquainted with markdown! :-)

**Pull request closed without merging**